sdks/python/apache_beam/io/external/xlang_kinesisio_it_test.py

```diff
@@ -267,10 +267,10 @@ def create_stream(self, stream_name):
         )
         time.sleep(2)
         break
-      except:  # pylint: disable=bare-except
+      except Exception as e:
         if i == retries - 1:
           logging.error('Could not create kinesis stream')
-          raise
+          raise e
```

Comment:
Raised the active exception at this point.

Reply:
lgtm